tutorials/reset_simulation.md

```diff
@@ -4,6 +4,11 @@ The Reset Gazebo transport API is exposed to allow resetting simulation to time
 It's possible to call this API using the command line or through the GUI.
 In addition to the API, we have also expanded the simulation system API with a Reset interface.
 
+To repeat this demo, run the `default` world:
+```bash
+gz sim default.sdf
```

Comment:
Let me know what the right world is to repeat the gif at the bottom. I would also update the thing later about `default` world, because I think it should just match the tutorial so others can try it.

Reply:
The gif in this tutorial is of `rolling_shapes.sdf`

Reply:
Fixed in https://github.com/gazebosim/gz-sim/pull/2383/commits/54b25fed16678b789aef8f3f00f0a149b1ef9f43